src/main/java/edu/kis/powp/jobs2d/command/manager/CommandManager.java

```diff
@@ -70,6 +70,22 @@ public synchronized DriverCommand getCurrentCommand() {
         return currentCommand;
     }
 
+    public synchronized String getStatisticCommand() {
+        if (currentCommand != null) {
+            CommandStatisticVisitor statisticVisitor = new CommandStatisticVisitor();
+            currentCommand.accept(statisticVisitor);
+
+            int totalSubCommandsCount = statisticVisitor.getOperateToCount() + statisticVisitor.getSetPositionCount();
+            int operateToCommandsCount = statisticVisitor.getOperateToCount();
+            double totalCommandLength = statisticVisitor.getTotalLength();
+
+            return "Number of sub-commands used: " + totalSubCommandsCount + "\nNumber of operateTo commands used: " +
+                    operateToCommandsCount + "\nTotal command length: " + totalCommandLength;
+        } else {
+            return "No statistics to display";
+        }
```

Comment:
CommandManager is not the right place for preparing such string. This method should not dependent of "currentCommand". Place it somewhere else (maybe, separate class in command/visitor package?)